src/scf-release/jobs/get-kubectl/spec

```diff
@@ -0,0 +1,11 @@
+---
+name: get-kubectl
+
+description: This job exists only to ensure the presence of the lubetcl binary in the role referencing it.
```

Comment:
Typo: `lubetcl`

Reply:
Done with commit 158b9b51fe310632299747b6a4e80eef87d80457

Reply:
And fully fixed with commit a381e6d3903a823a22499bf4fad50b3697b1c5eb